### ISSUE TITLE
Please allow Collapse settings to be overridden through the default.less settings

### ISSUE THREAD

**Issue by davidhatten** [ant-design/ant-design#9923]
- [X] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
I want to override the padding, margins, and other styles of various elements of the Collapse component. Doing this via css classes is rather cumbersome, as I have to specify the entire chain of classes to override styles. For example, if I want to change the style on a Collapse Panel header, I have to use this selector:

`.ant-collapse .ant-collapse-item .ant-collapse-header`

I think the right way to do this is like other component, and provide settings in the `default.less` file to override them in a local theme.

### What does the proposed API look like?
After looking at how the other component settings work, something like:
 
```
@collapse-header-padding
@collapse-item-padding
@collapse-item-background-color
```

For me, I only want padding and selected color options, but I'm not sure what other users may find useful.

**Comment by ddcat1115** [comment]
You're welcome to raise a PR to add some less variables~

**Comment by davidhatten** [comment]
I have raised a PR and I believe my changes are valid, but I'm catching some kind of cascading lint error across a bunch of files I haven't touched. I don't know, maybe the error makes more sense to you, but I can't make heads or tails of it.

**Comment by davidhatten** [comment]
This has been fixed by PR #9943. I am closing the issue. Thank you.
